CONTRIBUTING.md

```diff
@@ -1,25 +1,26 @@
 We have [good first issues][good-first-issue] for new contributors and [help wanted][help-wanted] issues for our other contributors.
 
-* `good first issue` has extra information to help you make your first contribution.
```

Comment:
There seem to be a lot of changes in common with the import renaming PR. Probably best to merge that one first and then rebase this?

Reply:
Yes, as noted in the PR description. Technically this could be merged separately but I wanted to see that tests still worked.